xchange-coinmate/src/main/java/org/knowm/xchange/coinmate/dto/trade/CoinmateOrderHistoryEntry.java

```diff
@@ -58,4 +92,72 @@ public BigDecimal getOriginalAmount() {
   public String getStatus() {
     return status;
   }
+
+  public BigDecimal getStopPrice() {
+    return stopPrice;
+  }
+
+  public boolean isTrailing() {
+    return trailing;
+  }
+
+  public Long getTrailingUpdatedTimestamp() {
+    return trailingUpdatedTimestamp;
+  }
+
+  public BigDecimal getOriginalStopPrice() {
+    return originalStopPrice;
+  }
+
+  public BigDecimal getMarketPriceAtLastUpdate() {
+    return marketPriceAtLastUpdate;
+  }
+
+  public BigDecimal getMarketPriceAtOrderCreation() {
+    return marketPriceAtOrderCreation;
+  }
+
+  public boolean isHidden() {
+    return hidden;
+  }
+
+  public BigDecimal getAvgPrice() {
+    return avgPrice;
+  }
+
+  public String getStopLossOrderId() {
+    return stopLossOrderId;
+  }
+
+  public String getOriginalOrderId() {
+    return originalOrderId;
+  }
+
+  public String getOrderTradeType() {
+    return orderTradeType;
+  }
+
+  @Override
+  public String toString() {
+    return "CoinmateOrderHistoryEntry{" +
+        "id=" + id +
+        ", timestamp=" + timestamp +
+        ", type='" + type + '\'' +
+        ", price=" + price +
+        ", remainingAmount=" + remainingAmount +
+        ", originalAmount=" + originalAmount +
+        ", status='" + status + '\'' +
+        ", orderTradeType='" + orderTradeType + '\'' +
+        ", stopPrice=" + stopPrice +
+        ", trailing=" + trailing +
+        ", trailingUpdatedTimestamp=" + trailingUpdatedTimestamp +
+        ", originalStopPrice=" + originalStopPrice +
+        ", marketPriceAtLastUpdate=" + marketPriceAtLastUpdate +
+        ", marketPriceAtOrderCreation=" + marketPriceAtOrderCreation +
+        ", hidden=" + hidden +
+        ", avgPrice=" + avgPrice +
+        ", stopLossOrderId=" + stopLossOrderId +
+        ", originalOrderId=" + originalOrderId +
+        '}';
+  }
```

Comment:
those toString's seam not needed

https://github.com/knowm/XChange/issues/2966

Reply:
@walec51 removed